Cargo.toml

```diff
@@ -17,3 +17,65 @@ edition = "2021"
 keywords = ["configuration", "language", "nix", "nickel"]
 repository = "https://github.com/tweag/nickel"
 homepage = "https://nickel-lang.org"
+readme = "README.md"
+
+[workspace.dependencies]
+nickel-lang-core = { path = "./core", default-features = false }
```

Comment:
We discussed that orally, but we might need to set a specific version - in addition to `path` - for releasing on crates.io.